### PR TITLE
Feature 91. Fix 'Reset button'. narek

### PR DESCRIPTION
**Pull request by narekacy**
This should work, but I have no idea why `dropdown-group.js` line 14 is underlined like undefined. 

Due to React.js we should use one state in order to manage our component. Unfortunately, this doesn't work / or I'm doing something wrong. 
By the way: lines 42 and 61 in `dropdown-group.js` are commented due to eslint rules. You'll need to use it while reviewing.

Check this code, and give me a feedback. please.
Thank you in adnvance.

### CODE REVIEW

File: client/src/components/dropdown-group.js [hunk 1]
```diff
@@ -23,6 +39,9 @@ class DropdownGroup extends React.Component {
           items={eventsType}
           homePath="events"
           path="events/categories/"
+          {/* eslint-disable-next-line no-undef */}
+          categories={categories} // here we pass our empty array (or updated later)
```
Comment (vitaliimak) [inline client/src/components/dropdown-group.js:43]: You don't have variable `categories` in function render.
`const { categories } = this.state;`

Comment (narekacy) [reply client/src/components/dropdown-group.js:43]: Thanks a lot

**Comment by vitaliimak** [comment]
Your branch is called `feature_number` but it must be `fix_number` 
I think it isn't important right now, but you should more attentive

File: client/src/components/dropdown-group.js (hunk 1 above)
Comment (narekacy) [reply client/src/components/dropdown-group.js:43]: Hmm.. but we can't consider `categories` 'used' if we pass it as a prop? )

Comment (narekacy) [reply client/src/components/dropdown-group.js:43]: It seems I answered my question :) It works. Thank you

**Comment by narekacy** [comment]
Thank you

**Pull request closed without merging**